src/packages.jl

```diff
@@ -3,10 +3,11 @@ module Packages
 
 import Downloads
 import Pidfile
+import Scratch: @get_scratch!
 import ...GAP: disable_error_handler, Globals, GapObj, replace_global!, RNamObj, Wrappers
 import ...GAP: gap_pkg_jlls, Compat, GAP_VERSION, GAP_jll, GAP_lib_jll
 
-gap_packages_rootdir() = joinpath(Base.DEPOT_PATH[1], "gaproot", "v$(GAP_VERSION.major).$(GAP_VERSION.minor)")
+gap_packages_rootdir() = @get_scratch!("v$(GAP_VERSION.major).$(GAP_VERSION.minor)")
```

Comment:
```suggestion
gap_packages_rootdir() = @get_scratch!("gap_packagedir_v$(GAP_VERSION.major).$(GAP_VERSION.minor)")
```
I think it would make sense to somehow name this. Two benefits:
1. you are able to find this inside your `~/.julia/scratchspaces` with an appropriate `find` call
2. the same folder also contains the gaproots (at least until https://github.com/oscar-system/GAP.jl/pull/1162)